src/components/Root.js

```diff
@@ -0,0 +1,15 @@
+import { Outlet } from 'react-router-dom';
+import MainNavigation from './mainNavigation';
+import classes from '../Root.module.css';
+
+function RootLayout() {
+  return (
+    <>
+      <MainNavigation />
+      <main className={classes.content}>
+        <Outlet />
+      </main>
```

Comment:
Well done here as well I want just to mention that the content here is not verticaly centered even with basic styling its very important to correctly align your element in the page.

![Capture d’écran du 2023-07-19 11-13-35](https://github.com/melkamu12/math-magicians/assets/59707859/0eb8c9f1-7b8c-4678-894b-7ba5655cddcf)